packages/checkbox-react/src/CheckboxGroup.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { ReactNode } from "react";
+import { FieldGroup } from "@fremtind/jkl-field-group-react";
+
+interface Props {
+    children: ReactNode;
+    legend: string;
+    className?: string;
+}
+
+export function CheckboxGroup({ legend, className, children }: Props) {
```

Comment:
Imo CheckboxGroup should have the same public API as RadioButtonGroup.
Instead of taking children (which must be Checkboxes) as props, it should take an array of strings which should map to Checkboxes.